examples/docker/pixi.toml

```diff
@@ -12,6 +12,9 @@ python = ">=3.12"
 flask = ">=3.0.2,<3.1"
 gunicorn = ">=21.2.0,<21.3"
 
+[pypi-dependencies]
+docker-project = { path = ".", editable = true }
```

Comment:
doesn't this clash with `postinstall-production` below @tdejager?

Reply:
Hmm yes it will get overwriten probably. But the .pth files might incorrectly stay in the prefix. We need the #1092 to do it correctly.1